relm4-book/src/factory_advanced.md

```diff
@@ -59,6 +59,7 @@ The factory implementation is mostly the same, so we'll just have a look at what
 ### The widgets type
 
 Because we have four actions per counter now, we also need an additional box to store these buttons.
+To be able to provide the root widget via the `get_root` function we need add the box to the data structure.
```

Comment:
I think that sentence is good but to make it more clear I would write "widgets type" instead of "data structure" and "to store ... in" instead of "add ... to".